topi/python/topi/x86/conv2d.py

```diff
@@ -263,7 +262,7 @@ def conv_arg_to_workload(data, kernel, strides, padding, layout, out_dtype):
 @autotvm.task.dispatcher
 def conv2d_x86(data, kernel, strides, padding, layout, out_dtype):
     """x86 conv2d declaration."""
-    return conv_arg_to_workload(data, kernel, strides, padding, layout, out_dtype)
+    return _conv_arg_to_workload(data, kernel, strides, padding, layout, out_dtype)
 
 
 @conv2d_x86.register(["direct"])
```

Comment:
We can use `@autotvm.topi_register_compute(conv2d, 'cpu', 'direct')` and delete the function `conv2d_x86`